src/activate/activate.c

```diff
@@ -69,7 +69,7 @@ static int open_sockets(int *epoll_fd, bool accept) {
 
         /* Close logging and all other descriptors */
         if (arg_listen) {
-                int except[3 + n];
+                int except[(size_t)(3 + n)];
```

Comment:
Why do not you drop VLA?
Also, instead of using `3`, I'd like to use `SD_LISTEN_FDS_START`.

Reply:
> Why do not you drop VLA?

Is there a better way I'm not aware of?

> Also, instead of using `3`, I'd like to use `SD_LISTEN_FDS_START`.

Good point, fixed, thanks!